driver-core/src/main/com/mongodb/internal/operation/CommandWriteOperation.java

```diff
@@ -1,62 +0,0 @@
-/*
- * Copyright 2008-present MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.mongodb.internal.operation;
-
-import com.mongodb.internal.async.SingleResultCallback;
-import com.mongodb.internal.binding.AsyncWriteBinding;
-import com.mongodb.internal.binding.WriteBinding;
-import org.bson.BsonDocument;
-import org.bson.codecs.Decoder;
-
-import static com.mongodb.assertions.Assertions.notNull;
-import static com.mongodb.internal.operation.CommandOperationHelper.executeCommand;
-import static com.mongodb.internal.operation.CommandOperationHelper.executeCommandAsync;
-
-/**
- * An operation that executes an arbitrary command that writes to the server.
- *
- * @param <T> the operations result type.
- * @since 3.0
- */
-public class CommandWriteOperation<T> implements AsyncWriteOperation<T>, WriteOperation<T> {
```

Comment:
This class was unused, and once I ditched it a lot of the CommandOperationHelper methods were no longer used.